packages/migration/src/migrations/test/v0034_markVideoAsExternal.js

```diff
@@ -0,0 +1,837 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import markVideoAsExternal from '../v0034_markVideoAsExternal';
+
+describe('markVideoAsExternal', () => {
+  it('should migrate 3P video to external', () => {
+    expect(
+      markVideoAsExternal({
+        _test: 'story',
+        pages: [
+          {
+            _test: 'page1',
+            elements: [
+              {
+                _test: 'element1',
+                type: 'square',
+                x: 10,
+                y: 20,
+                width: 100,
+                height: 200,
+              },
+              {
+                _test: 'element2',
+                type: 'image',
+                resource: {
+                  type: 'image',
+                  mimeType: 'image/png',
+                  src: 'https://example.com/image.png',
+                  width: 265,
+                  height: 527,
+                  isExternal: true,
```

Comment:
```suggestion
```

Need to remove this so we can actually test that `isExternal: true` is added by the migration